Bump the reported PHP version to 7.4.34 to mark the security backports;
no upstream 7.4.34 release exists.

--- a/configure.ac
+++ b/configure.ac
@@ -17,7 +17,7 @@
 dnl ----------------------------------------------------------------------------
 
 AC_PREREQ([2.68])
-AC_INIT([PHP],[7.4.33],[https://bugs.php.net],[php],[https://www.php.net])
+AC_INIT([PHP],[7.4.34],[https://bugs.php.net],[php],[https://www.php.net])
 AC_CONFIG_SRCDIR([main/php_version.h])
 AC_CONFIG_AUX_DIR([build])
 AC_PRESERVE_HELP_ORDER
--- a/main/php_version.h
+++ b/main/php_version.h
@@ -2,7 +2,7 @@
 /* edit configure.ac to change version number */
 #define PHP_MAJOR_VERSION 7
 #define PHP_MINOR_VERSION 4
-#define PHP_RELEASE_VERSION 33
+#define PHP_RELEASE_VERSION 34
 #define PHP_EXTRA_VERSION ""
-#define PHP_VERSION "7.4.33"
-#define PHP_VERSION_ID 70433
+#define PHP_VERSION "7.4.34"
+#define PHP_VERSION_ID 70434
